flags/test.ts

```diff
@@ -197,37 +197,57 @@ Deno.test("booleanParsingTrueLike", function (): void {
 });
 
 Deno.test("booleanNegationAfterBoolean", function (): void {
-  const parsed = parse(["--foo", "--no-foo"], { boolean: ["foo"] });
+  const parsed = parse(["--foo", "--no-foo"], {
+    boolean: ["foo"],
+    negatable: ["foo"],
+  });
   assertEquals(parsed.foo, false);
 
-  const parsed2 = parse(["--foo", "--no-foo", "123"], { boolean: ["foo"] });
+  const parsed2 = parse(["--foo", "--no-foo", "123"], {
+    boolean: ["foo"],
+    negatable: ["foo"],
+  });
   assertEquals(parsed2.foo, false);
 });
 
 Deno.test("booleanAfterBooleanNegation", function (): void {
-  const parsed = parse(["--no--foo", "--foo"], { boolean: ["foo"] });
+  const parsed = parse(["--no--foo", "--foo"], {
```

Comment:
nit: Not an error of this change, but I think this `"--no--foo"` should be `"--no-foo"`.

`parsed` here looks like `{ _: [], "no--foo": true, foo: true }`, and that doesn't check what the test name suggests.

Reply:
fixed